src/foundry/foundry.js/pointSources/soundSource.d.ts

```diff
@@ -0,0 +1,42 @@
+interface SoundSourceData extends PointSource.Data {
+  x: number;
```

Comment:
missing doc

Reply:
This is inherited from `PointSource.Data`.